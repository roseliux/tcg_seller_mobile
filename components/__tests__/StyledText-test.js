import renderer, { act } from 'react-test-renderer';

import { MonoText } from '../StyledText';

it(`renders correctly`, () => {
  let tree;

  act(() => {
    tree = renderer.create(<MonoText>Snapshot test!</MonoText>);
  });

  expect(tree.toJSON()).toMatchSnapshot();
});
