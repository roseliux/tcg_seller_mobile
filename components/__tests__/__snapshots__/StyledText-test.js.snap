// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`renders correctly 1`] = `
<Text
  style={
    [
      {
        "color": "#000",
      },
      [
        undefined,
        {
          "fontFamily": "SpaceMono",
        },
      ],
    ]
  }
>
  Snapshot test!
</Text>
`;
